src/test/java/io/burt/athena/result/S3ResultTest.java

```diff
@@ -30,10 +30,7 @@
 import java.util.concurrent.TimeoutException;
 
 import static io.burt.athena.support.GetQueryResultsHelper.createColumn;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please configure your editor to not change imports automatically (I suspect the default IDEA setting that replaces five imports by a star)